src/System.Net.Requests/tests/HttpWebRequestTest.netstandard1.7.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Those are TargetGroup pivots so they are just another example. 

>What is the benefit for testing netstandard1.3 and netstandard1.7 at this point? Isn't 1.3 a subset of 1.7?

The benefit is so we can still run the majority of tests on platforms that don't yet support netstandard1.7, example UWP. However with that said we are in the process of updating the corefx engineering process (in dev/eng branch) in which case we will essentially only build for the latest version of each target framework (i.e. latest netcoreapp, netstandard, uap, netfx, etc) and once that happens we will no longer have a netstandard1.3 configuration.